cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -96,7 +95,7 @@ template <
     typename            OutputIteratorT,    ///< Random-access output iterator type for writing scan outputs \iterator
     typename            ScanTileStateT,     ///< Tile status interface type
     typename            ScanOpT,            ///< Binary scan functor type having member <tt>T operator()(const T &a, const T &b)</tt>
-    typename            InitValueT,         ///< Initial value to seed the exclusive scan (cub::NullType for inclusive scans)
+    typename            InputValueT,        ///< Initial value to seed the exclusive scan (cub::NullType for inclusive scans)
```

Comment:
I wouldn't change the name of this -- I think `InitValueT` better describes what this represents.

Reply:
fixed